src/FakeItEasy/Core/AssemblyExtensions.cs

```diff
@@ -14,7 +14,16 @@ public static bool ReferencesFakeItEasy(this Assembly assembly)
         {
             Guard.AgainstNull(assembly, nameof(assembly));
 
+#if FEATURE_REFLECTION_GETASSEMBLIES
             return assembly.GetReferencedAssemblies().Any(r => r.FullName == TypeCatalogue.FakeItEasyAssembly.FullName);
+#else
+            var fakeItEasyLibraryName = TypeCatalogue.FakeItEasyAssembly.GetName().Name;
+            var context = Microsoft.Extensions.DependencyModel.DependencyContext.Default;
+
+            var runtimeLib = context.RuntimeLibraries

```

Comment:
Why RuntimeLibraries instead of CompileTime? Sorry, I know nothing. But I'd expect that if an assembly references FakeItEasy it would do so at compile time. (Perhaps this is a silly question, but docs are hard to come by - I couldn't learn anything about `DepenencyContext`.)